components/cdc/src/delegate.rs

```diff
@@ -80,9 +89,10 @@ pub struct Downstream {
     // The IP address of downstream.
     peer: String,
     region_epoch: RegionEpoch,
-    sink: Option<BatchSender<CdcEvent>>,
+    sink: Option<RateLimiter<CdcEvent>>,
```

Comment:
Nit: `RateLimiter` is an implementation detail, not a sink, I prefer hiding the implementation detail inside `BatchSender`, though it's not a blocking issue.